### PR TITLE
Add some security HTTP Headers

### PR DESCRIPTION
**Pull request by elnappo**
Adds the following HTTP header (set by Django):

* X-XSS-Protection 
* X-Content-Type-Option
* Referrer-Policy

See #281

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
travis is unhappy.

**Comment by ThomasWaldmann** [comment]
please rebase -i on current master.

File: nsupdate/settings/base.py [hunk 1]
```diff
@@ -266,6 +268,9 @@
 LOGOUT_REDIRECT_URL = '/'
 
 X_FRAME_OPTIONS = 'DENY'  # for clickjacking middleware
+SECURE_BROWSER_XSS_FILTER = True
+SECURE_CONTENT_TYPE_NOSNIFF = True
+REFERRER_POLICY = 'strict-origin-when-cross-origin'
```
Comment (ThomasWaldmann) [inline nsupdate/settings/base.py]: why not 'same-origin'?

**Comment by elnappo** [comment]
Added commit https://github.com/nsupdate-info/nsupdate.info/pull/410/commits/9723a8490797d10500730953be628d51765c38b0

**Comment by ThomasWaldmann** [comment]
today i've update the prod site to master branch (and also updated dependencies, but still dj 1.11).

if this is finished, we can merge it and try it soon.

**Comment by elnappo** [comment]
Is ready to merge from my side. Should I add some additional header? CSP is rather complex and error prune.

**Comment by ThomasWaldmann** [comment]
Checking nginx now...

**Comment by elnappo** [comment]
Your'e right, we don't need `X-Forwarded-Host`. I removed it.

We also could add the Feature-Policy header:
```
proxy_set_header Feature-Policy "geolocation 'none'; midi 'none'; notifications 'none'; push 'none'; sync-xhr 'none'; microphone 'none'; camera 'none'; magnetometer 'none'; gyroscope 'none'; speaker 'none'; vibrate 'none'; fullscreen 'none'; payment 'none'; xr 'none'; usb 'none'; autoplay 'none'; legacy-image-formats 'none'; wake-lock 'none'";
```
https://github.com/w3c/webappsec-feature-policy
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy
https://scotthelme.co.uk/a-new-security-header-feature-policy/

**Comment by ThomasWaldmann** [comment]
Our feature policy would read a bit like "fat free, sugar free, gluten free" printed on a bottle of mineral water.

But if it doesn't cause issues, why not have it...

**Comment by elnappo** [comment]
Yes, unfortunately there is no "deny all" parameter...

**Pull request merged**